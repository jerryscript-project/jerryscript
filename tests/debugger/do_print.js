// Copyright JS Foundation and other contributors, http://js.foundation
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

print("Hello world!");
print("A", "[", "", "]", 1.1e2, null, true, undefined);
print();
print("[A\nB", "C\nD", "E\nF\nG]");

var s = "";

for (i = 1; i <= 10; i++) {
  /* Translated from hungarian: life is beautiful */
  s += i + ": Az élet gyönyörű.\n";
}

/* Long string with non-ascii characters. */
print(s);
