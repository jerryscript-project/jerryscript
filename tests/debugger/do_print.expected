Connecting to: localhost:5001
Stopped at tests/debugger/do_print.js:15
(jerry-debugger) c
Hello world!
A [  ] 110 null true undefined

[A
B C
D E
F
G]
1: Az élet gyönyörű.
2: Az élet gyönyörű.
3: Az élet gyönyörű.
4: Az élet gyönyörű.
5: Az élet gyönyörű.
6: Az élet gyönyörű.
7: Az élet gyönyörű.
8: Az élet gyönyörű.
9: Az élet gyönyörű.
10: Az élet gyönyörű.

